dhcpv4/utils.go

```diff
@@ -0,0 +1,14 @@
+package dhcpv4
+
+// IsRequested function takes a DHCPv4 message and an OptionCode, and returns
+// true if that option is within the requested options of the DHCPv4 message.
+func IsRequested(pkt *DHCPv4, requested OptionCode) bool {
```

Comment:
we should call this function with a more obvious (despite longer) name. What about `IsOptionRequested` ?

Reply:
also wondering if this shouldn't be a method of `DHCPv4`

Reply:
Sounds good, I'll rename it `IsOptionRequested` and make it a method of `DHCPv4`.